types/js-worker-search/tslint.json

```diff
@@ -0,0 +1,6 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules" : {
+        "npm-naming": [true,{"mode":"code","errors":[["NeedsExportEquals",false]]}]
```

Comment:
Why is this needed?

Reply:
Good catch! This was from an earlier way of handling exports. Removed!